File: default_output/phoenix/butterbrezel/butterbrezel_migration.ex
defmodule FinalLocallity.Repo.Migrations.CreateButterBrezel do
  use Ecto.Migration

  def change do
    create table(:butter_brezel) do
      add :butter_brezel_uuid, :string
      add :age, :integer
      add :weight, :integer
      add :name, :string

      timestamps()
    end

  end
end

File: default_output/flutter/butterbrezel/butterbrezel_migration.dart
class Database
{
    void execute(String s) {}
}

class ButterBrezelMigration
{
    void createDatabase(Database db)
    {
        db.execute("CREATE TABLE ButterBrezel"
              +"butter_brezel_uuid TEXT, "
              +"age INTEGER, "
              +"weight INTEGER, "
              +"name TEXT"
            ")"
        );
    }
}
